tests/kubernetes/kubernetes_request_factory/test_pod_request_factory.py

```diff
@@ -62,7 +63,7 @@ def setUp(self):
                 'fsGroup': 2000,
             }
         )
-        self.maxDiff = None
+        self.max_diff = None
```

Comment:
```suggestion
```

Reply:
It doesn't work properly if it is written in a different way.

Reply:
Fixed.